src/backend/marsha/core/api/video.py

```diff
@@ -300,7 +300,7 @@ def start_live(self, request, pk=None):
         if video.live_state is None:
             return Response({"error": "Call initiate-live before starting a live"}, 400)
 
-        if video.live_state not in [defaults.IDLE, defaults.PAUSED]:
+        if video.live_state not in [defaults.IDLE, defaults.PAUSED, defaults.HARVESTED]:
```

Comment:
```suggestion
        if video.live_state not in [defaults.IDLE, defaults.PAUSED, defaults.HARVESTED]:
```
shouldn't it be STOPPED instead of PAUSED?

Reply:
Oh yes, thanks